README.md

```diff
@@ -1,41 +1,71 @@
+
+<p align="center">
+  <img src="/media/readme/settings_opt.png" width="20%" alt="A screenshot of mirrord's status bar menu in the VSCode UI, with 'Settings' highlighted">
+</p>
 
-mirrord reads its configuration from the following locations:
+mirrord can read its configuration from multiple places. For more details, see [the documentation on setting up the mirrord config](https://mirrord.dev/docs/using-mirrord/vscode-extension/#using-the-mirrord-config).
 
-1. Active config can be set for the whole workspace using the `selectActiveConfig` command or the link in the dropdown menu. If active config is set, mirrord always uses it.
-2. If active config is not set, mirrord searches process environment (specified in launch configuration) for `MIRRORD_CONFIG_FILE` variable. This path can use the `${workspaceFolder}` variable.
-3. If no config is specified, mirrord looks for a default project config file in the `.mirrord` directory with a name ending with `mirrord.{json,toml,yaml,yml}`. If there is no default config file, mirrord uses default configuration values for everything. If there are many candidates for the default config file, mirrord sorts them alphabetically and uses the first one.
+<!-- mirrord reads its configuration from the following locations:
+
+1. An active config can be set for the whole workspace using the `selectActiveConfig` command or the link in the status bar menu. If an active config is set, mirrord always uses it.
+2. If an active config is not set, mirrord searches the process environment (specified in the launch configuration) for the `MIRRORD_CONFIG_FILE` variable. This path can use the `${workspaceFolder}` variable.
+3. If no config is specified, mirrord looks for a default project config file in the `.mirrord` directory with a name ending with `mirrord.{json,toml,yaml,yml}`. If there is no default config file, mirrord uses default configuration values for everything. If there are multiple candidates for the default config file, mirrord sorts them alphabetically and uses the first one. -->
 
 You can use the `changeSettings` command or the link in the dropdown menu to quickly edit detected configs.
+
+## Helpful Links
+
+* [Official documentation for this extension](https://mirrord.dev/docs/using-mirrord/vscode-extension/)
+* [Official language-specific guides for debugging](https://metalbear.co/guides/)
+
+## Contributions, feature requests, issues and support
+
+* Feel free to join to our [Discord channel](https://discord.gg/metalbear) if you need help using mirrord, or if you encounter an issue while using the extension.
+* Check our open issues for [the VSCode extension](https://github.com/metalbear-co/mirrord-vscode/issues) and [mirrord's core code](https://github.com/metalbear-co/mirrord/issues), and 👍 react to any that you would like to see addressed.
+* Before submitting a pull request for new features, please discuss it with us first by opening an issue or a discussion.
```

Comment:
Maybe link to the contributing guide instead? I think this point is mentioned there (and if it's not then it should/will be)